src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo-link {
+  display: flex;
+  padding: 10px 0;
```

Comment:
1) remove padding: 10px 0; from logo-link
2) add padding to links at navigation
3) add align-items: center to your header
4) change display: flex; at logo-link to display: inline-block or bloc

Reply:
@Viktor-Kost if I change display: flex to display: block, there will appear additional 4px under IMG, what should I do to avoid it?

Reply:
what do you mean? I don't see any additional spaces
![image](https://user-images.githubusercontent.com/98764641/232719092-900aa614-0969-4463-b320-4b7f149c3396.png)
And for the future: If you have questions, ask in the chat. You will get answer much faster

Reply:
> what do you mean? I don't see any additional spaces ![image](https://user-images.githubusercontent.com/98764641/232719092-900aa614-0969-4463-b320-4b7f149c3396.png) And for the future: If you have questions, ask in the chat. You will get answer much faster

I`ve already fix it, but not like @Viktor-Kost said. I applied display: block not for <a>, and for <img> and that empty space disappeared)